### ISSUE TITLE
Is there an introduction to the use of data in the ERA5 section and similar sample_era5.ipynb?

### ISSUE THREAD

**Issue by hzkkun** [lzy12301/S3GM#2]

**Comment by lzy12301** [comment]
The use of ERA5 data is quite similar to the 2D case provided in this repo, which can be used with only slight modifications (e.g., change data_name, version, test data loading part, and hyperparameters). For the source data and checkpoint for ERA5, we now added the data and checkpoint for ERA5 in our zenodo repo as a new version in [10.5281/zenodo.14607274](https://zenodo.org/records/14607274).

**Comment by hzkkun** [comment]
I used the ERA5 data you gave and the test points and I didn't get the corresponding results

**Comment by lzy12301** [comment]
Can you describe the problem in detail?

**Comment by hzkkun** [comment]
I wrote a sample_era5.ipynb with reference to the sample_kol.ipynb you gave, I used the trained model and verification data you gave, and found that the loss exploded to more than 2e+20 during the reconstruction process, I don't know if there is a problem with the ipynb I wrote

**Comment by hzkkun** [comment]
I wrote a sample_era5.ipynb with reference to the sample_kol.ipynb you gave, just deleted some unwanted variables, I used the trained model and verification data you gave, and found that the loss exploded to more than 2e+20 during the reconstruction process, I don't know if there is a problem with the ipynb I wrote
![1](https://github.com/user-attachments/assets/151f1d44-ddef-44fb-8cf6-9f6b8d9184c0)

**Comment by hzkkun** [comment]
I see that you use the global mean and variance for the ERA5 data normalization in the dataset.py file in the trainer directory, but the normalization for the validation data is the mean and variance corresponding to the 4 features

**Comment by lzy12301** [comment]
I think you can adjust these following hyperparameters (e.g., try smaller values for *alpha_case*):
![image](https://github.com/user-attachments/assets/7ec551f9-a728-4615-a405-7b93162d54a8)

**Comment by lzy12301** [comment]
The following settings worked for me.
![image](https://github.com/user-attachments/assets/88c48d80-df4c-4991-b593-f624d05b2885)
